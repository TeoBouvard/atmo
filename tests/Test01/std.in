1
10
10
1000
2016-12-12
2017-02-02
5
