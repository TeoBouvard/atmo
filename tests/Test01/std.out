Importation des capteurs ... 1 capteurs importÃ©s
Importation des mesures ... 4 mesures importÃ©es

Menu :
 1 : ConnaÃ®tre la qualitÃ© de lâ€™air sur une zone donnÃ©e
 2 : Identifier les zones ayant une qualitÃ© de lâ€™air similaire
 3 : ConnaÃ®tre la qualitÃ© de l'air en un point donnÃ©
 4 : Identifier les capteurs dÃ©fectueux
 5 : Quitter


Latitude : Longitude : Rayon : Date de dÃ©but au format YYYY-MM-DD : Date de fin au format YYYY-MM-DD : 
Analyse en cours ... 

[1m[37mNombre de capteurs dans la zone : 1

[1m[36mMoyenne 03 : 68.5 Âµg/m3 | Sous indice 03 : 3
Moyenne NO2 : 99 Âµg/m3 | Sous indice NO2 : 4
Moyenne SO2 : 119 Âµg/m3 | Sous indice SO2 : 3
Moyenne PM10 : 17 Âµg/m3 | Sous indice PM10 : 3

[1m[31mIndice ATMO : 4[0m

Analyse terminÃ©e.

Menu :
 1 : ConnaÃ®tre la qualitÃ© de lâ€™air sur une zone donnÃ©e
 2 : Identifier les zones ayant une qualitÃ© de lâ€™air similaire
 3 : ConnaÃ®tre la qualitÃ© de l'air en un point donnÃ©
 4 : Identifier les capteurs dÃ©fectueux
 5 : Quitter

